incubator/virtualcluster/pkg/controller/virtualcluster/master_provisioner_native.go

```diff
@@ -272,25 +285,42 @@ func (mpn *MasterProvisionerNative) createPKI(vc *tenancyv1alpha1.VirtualCluster
 	}
 	caGroup.ETCD = etcdCAPair
 
+	clusterIP := ""
+	isClusterIP := cv.Spec.APIServer.Service != nil && cv.Spec.APIServer.Service.Spec.Type == v1.ServiceTypeClusterIP
```

Comment:
I suggest to move isClusterIP definition before line 80 so that the if statement in line 80 can be `if isClusterIP`.

Reply:
good catch, fixing